addons/cookoff/functions/fnc_getVehicleAmmo.sqf

```diff
@@ -0,0 +1,58 @@
+/*
+ * Author: PabstMirror
+ * Gets all magazines inside of a vehicle.
+ *
+ * Arguments:
+ * 0: Vehicle <Object>
+ *
+ * Return Value:
+ * 0: Ammo Array <ARRAY>
+ *      0: Magazine Classname <STRING>
+ *      1: Ammo Count <NUMBER>
+ * 1: Total Ammo Count <NUMBER>
+ *
+ * Example:
+ * [vehicle player] call ace_cookoff_fnc_getVehicleAmmo
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+params ["_vehicle"];
+TRACE_1("getVehicleAmmo",_vehicle);
+
+private _ammoToDetonate = [];
+private _totalAmmo = 0;
+
+// Get ammo from turrets
+if (_vehicle isKindOf "AllVehicles") then {
```

Comment:
good catch, 
I was originally using `magazinesAmmo` for turret ammo and `magazinesAmmoCargo` for cargo ammo
But for ReammoBoxs they returned the same thing, so it was doubling the count.

`magazinesAllTurrets` doesn't have that issue, so we don't need that if statement